### PR TITLE
Create Bitbucket getting started guide

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**

### CODE REVIEW

File: content/docs/start/bitbucket.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+           script:
+             - pip install -r requirements.txt
+             - python train.py
+
+             - cat metrics.txt > report.md
+             - cml-publish confusion_matrix.png --md >> report.md
+             - cml-send-comment report.md
+   ```
+
+3. In your text editor of choice, edit line 16 of `train.py` to `depth = 12`.
```
Comment (dacbd) [inline content/docs/start/bitbucket.md:38]:
```suggestion
3. In your text editor of choice, edit line 15 of `train.py` to `depth = 12`.
```

**Review by dacbd — CHANGES_REQUESTED** [review]
I believe this was something that user mentioned for this repo/ but on gitlab 
image

File: content/docs/start/bitbucket.md (hunk 1 above)
Comment (0x2b3bfa0) [reply content/docs/start/bitbucket.md:38]: [It's line 16 on the repository](https://bitbucket.org/iterative-ai/cml-base-case/src/32e60c3e0cc124e40b1681b9b7b0458f3a238474/train.py#lines-16) 🤔

**Comment by 0x2b3bfa0** [comment]
@dacbd, repositories were outdated, please pull again from `example-cml` and try again

**Review by dacbd — CHANGES_REQUESTED** [review]
image

I am explicitly following the instructions, I believe we need to first create the pipeline file and enable it before it runs, idk bb UI is a little weird both times I have had deviate from the getting started instructions to get it run.

**Comment by 0x2b3bfa0** [comment]
https://github.com/iterative/dvc/issues/2760#issuecomment-551952674

**Comment by 0x2b3bfa0** [comment]
> _we need to first create the pipeline file and enable it before it runs_

Despite the Bitbucket Pipelines screen suggesting a first manual run, I have been able of triggering it automatically without any further action. 🤔

**Comment by 0x2b3bfa0** [comment]
Argh! https://github.com/iterative/example_cml/pull/219

File: content/docs/start/bitbucket.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+   The following steps can all be done in the Bitbucket browser interface.
+   However, to follow along the commands, we recommend cloning your fork to your
+   local workstation:
+
+   ```bash
+   git clone https://bitbucket.org/<your-username>/example-cml
+   ```
+
+2. To create a CML workflow, copy the following into a new file,
```
Comment (dacbd) [inline content/docs/start/bitbucket.md:20]:
```suggestion
2. To create a CML workflow, copy the following into a new file on your main branch,
```
lets either add this small caveat or add instructions to add the pipeline file first to the main branch.

In my testing, it doesn't work if its part of the pr which following these steps explicitly it is.

**Review by dacbd — CHANGES_REQUESTED** [review]
image

Do we need to include some form of PAT instructions like with the GitLab version of this?

**Comment by 0x2b3bfa0** [comment]
Yes. :see_no_evil: I'd avoid duplication and just link to https://cml.dev/doc/self-hosted-runners?tab=Bitbucket#personal-access-token.

**Comment by dacbd** [comment]
followed token instructions: 
image
🙃

**Comment by 0x2b3bfa0** [comment]
@dacbd, unable to reproduce. 🤔

**Comment by 0x2b3bfa0** [comment]
@dacbd, sounds like this old friend: 
* https://stackoverflow.com/a/66263032/4654476
* https://cml.dev/doc/ref/send-comment#bitbucket
Worth adding to the laundry list of tutorial steps?

**Comment by dacbd** [comment]
> @dacbd, sounds like this old friend:
> 
> * https://stackoverflow.com/a/66263032/4654476
> * https://cml.dev/doc/ref/send-comment#bitbucket
>   Worth adding to the laundry list of tutorial steps?

yes, I think that people are likely to walk through the getting started steps before reading any specific documentation.

**Review by dacbd — APPROVED** [review]
who uses bitbucket these days anyway, if they do they are probably better at navigating the UI than I am 🙃 🙉

**Pull request merged**